subsys/zigbee/Kconfig

```diff
@@ -6,6 +6,7 @@
 
 menuconfig ZIGBEE
 	bool "Enable Zigbee stack"
```

Comment:
```suggestion
	bool "Enable Zigbee stack" [EXPERIMENTAL]
```

Reply:
Experimental, or deprecated? And should the tag be inside the quotation marks?